views/books.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  <input class="pull-right" type="search" ng-model="q" placeholder="filter books..." aria-label="filter books" />
+  <form ng-show="add" ng-submit="booksIndexCtrl.createBook(); add = false">
+    <input type="text" class='form-control' ng-model="booksIndexCtrl.newBook.title" placeholder='New book title' >
```

Comment:
Should use labels rather than placeholders to inform people about what to put in boxes; recall that placeholders aren't guaranteed to be shown on all devices.